packages/circuit-ui/components/NotificationToast/NotificationToast.docs.mdx

```diff
@@ -5,16 +5,11 @@ import { NotificationToast } from '@sumup/circuit-ui';
 
 <Status.Stable />
 
+The `NotificationToast` component provides quick status messages to the user. A toast can follow a user action (e.g. submitting a form) or it can appear without having been triggered explicitly (e.g. a toast message "You made a new sale" showing real time sales in an app).
```

Comment:
```suggestion
The `NotificationToast` component provides quick status messages to the user. A toast can follow a user action (e.g. submitting a form) or it can appear without having been triggered explicitly (e.g. a toast message "You made a new sale" showing real-time sales in an app).
```